app/static/src/scss/assessment_flow.scss

```diff
@@ -0,0 +1,24 @@
+.grey-background{
+    background-color: #f3f2f1;
```

Comment:
🤔 There is govuk sass variables you can use here. Please see https://sass-lang.com/documentation/variables and https://design-system.service.gov.uk/styles/colour/